Document that getHeadings() only applies to the MDX file, not childre… (#7409)
withastro/docs: src/content/docs/en/guides/markdown-content.mdx

});
```

:::note

> The headings returned by `getHeadings()` are only those in the Markdown and MDX file. For MDX you might use components that have their own headings, but these won't be included in what is returned by `getHeadings()`.

Reviewer: ```suggestion
`getHeadings()` only returns headings directly in the Markdown and MDX file itself. If an MDX file imports components that have their own headings,these will be included in what is returned by `getHeadings()`.
```

> `getHeadings()` only returns headings written directly in the Markdown and MDX file itself. If an MDX file imports components that have their own headings,these will not be included in what is returned by `getHeadings()`.

Reviewer: ```suggestion
`getHeadings()` only returns the headings written directly in the Markdown and MDX file itself. If an MDX file imports components that have their own headings, these will not be included in what is returned by `getHeadings()`.
```

> `getHeadings()` only returns the headings written directly in the Markdown and MDX file itself. If an MDX file imports components that have their own headings, these will not be included in what is returned by `getHeadings()`.

Reviewer: ```suggestion
`getHeadings()` only returns the headings written directly in a Markdown or MDX file itself. If an MDX file imports components that have their own headings, these will not be returned by `getHeadings()`.
```
